api/types/src/convert.rs

```diff
@@ -79,6 +80,24 @@ impl<'a, R: MoveResolverExt + ?Sized> MoveConverter<'a, R> {
         self.inner.view_resource(typ, bytes)?.try_into()
     }
 
+    pub fn try_into_resources_from_resource_group(
+        &self,
+        bytes: &[u8],
+    ) -> Result<Vec<MoveResource>> {
+        // Resource groups cannot be nested.
```

Comment:
Could you expand on this comment a bit, like why this callout matters here?

Reply:
+1
let's delete the comment or we need a long description of resource groups in all the snippets of code.